### PR TITLE
rename and add doctest 

### PR DESCRIPTION
**Pull request by shellhub**
@cclauss

### CODE REVIEW

File: maths/find_max.py [hunk 1]
```diff
@@ -1,16 +1,21 @@
-# NguyenU
```
Comment (cclauss) [inline maths/find_max.py]: This file is out of sync with `master`.  There are already doctests on this file:
https://github.com/TheAlgorithms/Python/blob/master/maths/find_max.py#L6

Comment (shellhub) [reply maths/find_max.py]: @cclauss Please review again!

**Comment by cclauss** [comment]
Please write a https://docs.python.org/3/library/timeit.html that proves that the new implementation is faster than the current implementation.  My bet is that the function and slice overhead means that it is not.

**Comment by shellhub** [comment]
> Please write a https://docs.python.org/3/library/timeit.html that proves that the new implementation is faster than the current implementation. My bet is that the function call and slice overhead means that it is not.

Did you mean below code implementation is slower ?
``` python
    max_num = nums[0]
    for i in range(1, len(nums)):
        if nums[i] > max_num:
            max_num = nums[i]
```

**Comment by cclauss** [comment]
Not sure but that is my _suspicion_ but a timeit test can tell us for sure.

**Comment by shellhub** [comment]
After I tested using 
``` python
def find_max(nums):
    max_num = nums[0]
    for i in range(1, len(nums)):
        if nums[i] > max_num:
            max_num = nums[i]
    return max_num
print(timeit.timeit("find_max([2, 4, 9, 7, 19, 94, 5])", setup="from __main__ import find_max", number=10000))
time: 0.022231400000000002
```

``` python
def find_max(nums):
    max = nums[0]
    for x in nums:
        if x > max:
            max = x
    return max
print(timeit.timeit("find_max([2, 4, 9, 7, 19, 94, 5])", setup="from __main__ import find_max", number=10000))
time: 0.007430200000000005
```

From the result, You are right. Thanks

**Comment by cclauss** [comment]
This is awesome work.  I think we should fill this file with as many different implementations of __find_max()__ as we can think of with timeit tests for each one in an effort to find that fastest pure Python implementation.

How does `print(timeit.timeit("max([2, 4, 9, 7, 19, 94, 5])", number=10000))` perform using the Python builtin?

**Comment by shellhub** [comment]
> print(timeit.timeit("max([2, 4, 9, 7, 19, 94, 5])", number=10000))

`0.004652999999999997` , Very fast

File: maths/find_min.py [hunk 2]
```diff
@@ -1,17 +1,21 @@
-"""Find Minimum Number in a List."""
+def find_min(nums):
+    """
+    Find Minimum Number in a List
+    :param nums: contains elements
+    :return: max number in list
 
+    >>> find_min([1, 3, 5, 7, 9]) == 1
```
Comment (cclauss) [inline maths/find_min.py]: Let's use the same tests data as find_max.py uses.

Comment (shellhub) [reply maths/find_min.py]: Updated. Please review again

**Pull request merged**